### ISSUE TITLE
Replace copyright headers in source files

### ISSUE THREAD

**Issue by haskellcamargo** [quack/quack#130]
Beginner friendly!

Quack has grown and no longer is the work of one single duck :baby_chick:, it is the work of a community.

We need to replace the comment copyright headers from:

```diff
 /**
  * Quack Compiler and toolkit
- * Copyright (C) 2016 Marcelo Camargo <marcelocamargo@linuxmail.org> and
- * CONTRIBUTORS.
```

To:

```diff
 /**
  * Quack Compiler and toolkit
+ * Copyright (C) 2015-2017 Quack and CONTRIBUTORS
```

- This should be applied across all files that match a copyright header (including C++ under `node/`)
- The year must be updated to 2015-2017

If you are going to get this task, comment bellow to avoid duplicated work!
I :heart: you all!

**Comment by rafaelcn** [comment]
I'm doing it! 😄

**Comment by luizperes** [comment]
Hi @haskellcamargo, wouldn’t it be better 2015~2017, instead of 2017?

**Comment by haskellcamargo** [comment]
@luizperes Yeah, I'll edit!

**Comment by rafaelcn** [comment]
I've just ran a regex to replace everything with just 2017. Seems like I'll have to redo 👅

**Comment by rafaelcn** [comment]
By the way, I couldn't find `interop/`. Could you please point to where it is @haskellcamargo?

**Comment by haskellcamargo** [comment]
Ah, sorry! It was `node/`. I forgot I renamed before pushing!

**Comment by rafaelcn** [comment]
No problem!

**Comment by haskellcamargo** [comment]
The source there still has no header, but we can add by copying from some `.php` from Quack.

**Comment by rafaelcn** [comment]
Just done it, opening a PR 👍

**Comment by rafaelcn** [comment]
This should be closed as per fixed on #131

**Comment by haskellcamargo** [comment]
Closed by #131
